scout/server/blueprints/clinvar/views.py

```diff
@@ -116,3 +117,27 @@ def clinvar_download_csv(submission, csv_type, clinvar_id):
         mimetype="text/csv",
         as_attachment=True,
     )
+
+
+@clinvar_bp.route("/<submission>/download/json/<clinvar_id>", methods=["GET"])
+def clinvar_download_json(submission, clinvar_id):
+    """Download a json for a clinVar submission"""
+
+    code, conversion_res = controllers.json_api_submission(submission_id=submission)
+
+    if code in [200, 201]:
+        # Write temp CSV file and serve it in response
+        tmp_json = NamedTemporaryFile(mode="a+", prefix=clinvar_id, suffix=".json")
```

Comment:
The suffix doesn't seem to stick when ou download the file..?

<img width="707" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/546574c1-087e-4eb4-9043-5e7217824dc2">

Reply:
This was solved by including .json in the name of the file. Have been tested both using Safari (which had the extension also before the change) and Chrome. Now seems to be working for both browsers.

Reply:
Yes, it works for me as well now:

<img width="646" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/ce7795fc-557b-4c7e-9870-765755b3b143">